src/logic/calculate.js

```diff
@@ -0,0 +1,125 @@
+import operate from './operate';
```

Comment:
- [ ] In this project, we are expected to concentrate on React events, that is why the logic files are provided for us. Your `calculate.js` file does not match the provided file [here](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/code-samples/calculate.js). Kindly fix this.

Reply:
Done!